floris/tools/floris_interface.py

```diff
@@ -838,6 +824,81 @@ def get_farm_AEP(
 
         return aep
 
+    def get_farm_AEP_with_wind_data(
+        self,
+        wind_data,
+        cut_in_wind_speed=0.001,
+        cut_out_wind_speed=None,
+        yaw_angles=None,
+        turbine_weights=None,
+        no_wake=False,
+    ) -> float:
+        """
+        Estimate annual energy production (AEP) for distributions of wind speed, wind
+        direction, frequency of occurrence, and yaw offset.
+
+        Args:
+            freq (NDArrayFloat): NumPy array with shape (n_findex)
```

Comment:
Double check that this docstring is correct and not a remnant of copy/paste. There's no `freq` in the arguments to this function.

Reply:
Updated, thanks